common/reviews/api/rush-lib.api.md

```diff
@@ -740,6 +740,8 @@ export interface _IPnpmOptionsJson extends IPackageManagerOptionsJsonBase {
     alwaysInjectDependenciesFromOtherSubspaces?: boolean;
     autoInstallPeers?: boolean;
     globalAllowedDeprecatedVersions?: Record<string, string>;
+    globalCatalog?: Record<string, string>;
+    globalCatalogs?: Record<string, Record<string, string>>;
```

Comment:
since `globalCatalog` is equivalent to `globalCatalogs.default`, what if we only supported `globalCatalogs` to keep the config definition and parsing simpler?